README.md

```diff
@@ -49,6 +49,21 @@ $.ajax({
     });
 ```
 
+## Allow default to all ajax requests
+
+In this example, there is default retry to all ajax requests, for 5 seconds between retries
+
+      $.ajaxSettings.
+            shouldRetry= function() {
+                // Always retry this request with a delay of 250ms between retries
+                return $.Deferred(function(dfr) {
+                    setTimeout(function() {
+                        dfr.resolve(true);
+                    }, 5000);
+                }).promise();
+            }
```

Comment:
The indentation of this should be such that it lines up with the `$`, and we're missing a `;` after the closing brace

Reply:
Fixed